src/templates/category.tsx

```diff
@@ -19,6 +19,11 @@ const Category = ({ data }) => {
     return null;
   }
 
+  let plugins = data.plugins;
+  if (category.displayName === "All Plugins") {
+    plugins = data.allPlugins;
+  }
```

Comment:
Do not use the displayName for matching.
```suggestion
  if (category.name === "all") {
    plugins = data.allPlugins;
  }
```